e2e_tests/tests/per_provider/normal_tests/aead.rs

```diff
@@ -213,3 +258,502 @@ fn aead_encrypt_ccm_encrypt_decrypt() {
 
     assert_eq!(&PLAINTEXT[..], plaintext.as_slice());
 }
+
+#[cfg(feature = "cryptoauthlib-provider")]
+#[test]
+fn aead_encrypt_ccm_encrypt_decrypt_shortened_tag() {
+    let key_name = String::from("aead_encrypt_ccm_encrypt_decrypt_shortened_tag");
+    let mut client = TestClient::new();
+    if !client.is_operation_supported(Opcode::PsaAeadEncrypt)
+        || !client.is_operation_supported(Opcode::PsaAeadDecrypt)
+    {
+        return;
+    }
+    client
+        .import_aes_key(
+            key_name.clone(),
+            KEY_DATA.to_vec(),
+            Aead::AeadWithShortenedTag {
+                aead_alg: AeadWithDefaultLengthTag::Ccm,
+                tag_length: 16,
```

Comment:
Isn't 16 the same as the default size?

Reply:
Default tag length is applicable only for Aead::AeadWithDefaultLengthTag(). When it comes to another API Aead::AeadWithShortenedTag() then the tag length must be explicitly specified.  Your comment refer to test aead_encrypt_ccm_encrypt_decrypt_shortened_tag which tests if the second API works well with cryptoauthlib provider.